src/formats/webm.moon

```diff
@@ -19,14 +19,16 @@ class WebmVP8 extends Format
 		colormatrix = mp.get_property_native("video-params/colormatrix")
 		if colormatrixFilter[colormatrix]
 			append(ret, {
-				"lavfi-colormatrix=#{colormatrixFilter[colormatrix]}:bt601"
+				MpvFilter("lavfi-colormatrix", {"@0": "#{colormatrixFilter[colormatrix]}", "@1": "bt601"})
```

Comment:
Just do `{"@0": colormatrixFilter[colormatrix], ...`

Also, what are those "@0" things? Is this just to enforce an ordering? As in, first, and second parameters? If so, remember that Lua doesn't guarantee order in dicts (see comment on `solveFilters`).